server/router.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const router = express.Router();
+
+const controller = require('./controller/index');
+
+router.get('/alerts/:city', controller.getAlert); //don't change path!
+router.get('/seeda', controller.seedAlerts);
+
+router.get('/disaster', controller.getPreparations);
```

Comment:
Not exactly the proper naming for RESTful endpoints, checkout hoe to be consistent with naming in the REST protocol.